test/unistd/isatty.c

```diff
@@ -21,7 +21,11 @@ int main() {
   assert(errno == EBADF);
 
   err = isatty(open("/dev/stdin", O_RDONLY));
+#ifdef NODERAWFS
+  assert(!err);
+#else
```

Comment:
Why is this needed? Perhaps add a comment?

Reply:
It was intended as a regression test, but it appears that `other.test_unistd_isatty` was already functioning correctly with `NODERAWFS`. Therefore, I reverted this with commit c39e5b14d3510f4fa673feeec3c76a85664a645f.